src/mongo/db/mongod_options.cpp

```diff
@@ -62,6 +62,11 @@ using std::string;
 MongodGlobalParams mongodGlobalParams;
 
 extern DiagLog _diaglog;
+extern int cursorTimeoutMillis;
```

Comment:
You are talking about different thing which is `ExportedServerParameter`. It really allows you map a variable (memory location) to be set via `setParameter` call. I'm talking about code correctness. What you have with `extern` won't allocate memory location for the variable, so the code won't link.